src/lib/incentives-calculation.ts

```diff
@@ -76,6 +77,14 @@ function calculateFederalIncentivesAndSavings(
       continue;
     }
 
+    if (
```

Comment:
Nit: Could we put the computation of the boolean in a named function to make it easier to understand? ie: `if (shouldExcludeV0(incentive.payment_methods[0]){ continue }` or something with a better name lol.